tools/msmetaenhancer/msmetaenhancer.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="msmetaenhancer" name="MSMetaEnhancer" version="0.1.0+galaxy0">
+    <description>annotate MS data</description>
+    <requirements>
+        <requirement type="package" version="0.1.0">msmetaenhancer</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh ${msmetaenhancer_python_cli}
+    ]]> </command>
+
+    <configfiles>
+        <configfile name="msmetaenhancer_python_cli">
+            python3 ${__tool_directory__}/msmetaenhancer_wrapper.py \
+            --input_file "$input_file" \
+            --output_file "$output_file" \
+            --services "$services"
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param label="Input spectra file" name="input_file" type="data" format="msp" help="Path to query spectra file in MSP format." />
+        <param name="services" type="select" label="Available services" multiple="true" display="checkboxes" optional="true">
+            <option value="PubChem">PubChem</option>
+            <option value="CTS">CTS</option>
+            <option value="CIR">CIR</option>
+            <option value="NLM">NLM</option>
+        </param>
+    </inputs>
+
+    <outputs>
+        <data label="Output spectra file" name="output_file" format="msp" />
```

Comment:
Maybe rename to "Enhanced ${input_file} to indicate what the tool was actually called on? Another option would be "MSMetaEnhancer on ${on_string}".

Reply:
Used second option in 7429f24.